tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -0,0 +1,115 @@
+"""Simple demo which displays constant 3D mesh."""
```

Comment:
This needs a build target (`//tensorboard/plugins/mesh:mesh_demo`, for
consistency with other plugins). You’ll need to set up the `trimesh`
dependency in `python.bzl` as we discussed previously.

Reply:
Thats what I was curious about yesterday. I'll add BUILD rule.